plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -257,7 +271,9 @@ private AzureStorageSettings(
         this.responseTimeout = responseTimeout;
         this.proxySettings = proxySettings;
     }
-
+    public boolean useManagedIdentityCredential() {
```

Comment:
Minor grammatical point - can we rename this to `usesManagedIdentityCredential`? Because useX = action, usesX = boolean check.

Reply:
Ooo, useful information, thank you arjun, fixed